src/schema/rules/files/raw/meg.yaml

```diff
@@ -3,19 +3,34 @@ meg:
   suffixes:
     - meg
   extensions:
-    - / # corresponds to BTi/4D data
-    - .ds/
+    # / corresponds to BTi/4D data
     - .json
-    - .fif
-    - .sqd
-    - .con
+    - [/, .ds/, .fif]
+    - [.sqd, .con]
+    - .ave # mentioned once in the appendix
+    # .raw and .raw.mhd work as a pair
     - .raw
-    - .ave
-    - .mrk
```

Comment:
.mrk seems to only exist for markers suffix

Reply:
yes, some context in this thread: https://github.com/bids-standard/bids-specification/issues/638#issuecomment-703451371 and in particular this reponse (i.e., why `.mrk` is *only* for the `markers` suffix, but `.sqd` can also be used for the brain data): https://github.com/bids-standard/bids-specification/pull/653#issuecomment-717503165